installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc

```diff
@@ -1,19 +1,32 @@
 :_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
 [id="preparing-to-install-on-ibm-z"]
 = Installation methods
-include::_attributes/common-attributes.adoc[]
 :context: preparing-to-install-on-ibm-z
 
 toc::[]
 
+You can install an {product-title} cluster on {ibm-z-name} and {ibm-linuxone-name} using a variety of different installation methods.
+Each method has qualities that can make them more suitable for different use cases, such as installing a cluster in a disconnected environment or installing a cluster with minimal configuration and provisioning.
+
+//moved prerequisites to ibm z cluster installation assemblies zvm, lpar, kvm
+
+[NOTE]
+====
+While this document refers only to {ibm-z-name}, all information in it also applies to {ibm-linuxone-name}.
+====
+
+[id="choosing-an-method-to-install-ocp-on-ibm-z"]
+== Choosing a method to install {product-title} on {ibm-z-title} or {ibm-linuxone-title}
+
 The {product-title} installation program offers the following methods for deploying a cluster on {ibm-z-name}:
 
 * *Interactive*: You can deploy a cluster with the web-based link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform[{ai-full}]. This method requires no setup for the installer, and is ideal for connected environments like {ibm-z-name}.
 
 * *Local Agent-based*: You can deploy a cluster locally with the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#preparing-to-install-with-agent-based-installer[Agent-based Installer]. It provides many of the benefits of the {ai-full}, but you must download and configure the Agent-based Installer first. Configuration is done with a command line interface (CLI). This approach is ideal for disconnected networks.
 // Tile on the console not available yet. once available add: the link:https://console.redhat.com/openshift/install/ibmz/agent-based[Agent-based Installer]
 
-* *Full control*: You can deploy a cluster on xref:../../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[infrastructure that you prepare and maintain], which provides maximum customizability. You can deploy clusters in connected or disconnected environments.
+* *Full control*: You can deploy a cluster on xref:../../installing/installing_ibm_z/upi/installing-ibm-z.adoc#installing-ibm-z[infrastructure that you prepare and maintain], which provides maximum customizability. You can deploy clusters in connected or disconnected environments.
```

Comment:
Thanks for bringing this up. Actually I think it's best to entirely remove the link, since the different UPI installations come further down on that page.